### PR TITLE
Move CLI Build to MSBuild 00024-160610

### PR DESCRIPTION
**Pull request by piotrpMSFT**
Part 2 of #4019

### CODE REVIEW

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

File: build.proj [hunk 1]
```diff
@@ -18,8 +18,6 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
     <CLIBuildFileName>$(RepoRoot)/build_projects/dotnet-cli-build/bin/dotnet-cli-build</CLIBuildFileName>
     <CLIBuildDll>$(CLIBuildFileName).dll</CLIBuildDll>
-    <BuildToolsDir>$(RepoRoot)/build_tools/</BuildToolsDir>
-    <ToolsDir>$(BuildToolsDir)/</ToolsDir>
```
Comment (piotrpMSFT) [inline build.proj]: the directories still exist, right? Why remove this?

**Comment by piotrpMSFT** [comment]
:Shipit:

File: build.proj (hunk 1 above)
Comment (eerhardt) [inline build.proj]: Because the properties aren't being used.  No reason to leave "junk" in the file.

**Pull request merged**